src/metabase/driver/postgres.clj

```diff
@@ -475,7 +522,9 @@
    (keyword "double precision")           :type/Float
    (keyword "time with time zone")        :type/Time
    (keyword "time without time zone")     :type/Time
-   (keyword "timestamp with timezone")    :type/DateTime
+   ;; TODO postgres also supports `timestamp(p) with time zone` where p is the precision
+   ;; maybe we should switch this to use `sql-jdbc.sync/pattern-based-database-type->base-type`
+   (keyword "timestamp with timezone")    :type/DateTimeWithTZ
```

Comment:
I'm 95% sure Postgres always calls `timestamp with time zone` a `timestamptz` in JDBC which is why the mapping for that is here 

https://github.com/metabase/metabase/pull/25698/files#diff-122d8f1486c5b97fd092fb98e7419b767c12d378bc6e668c16b1041cd965f9cbR512

At any rate `time zone` is two words in Postgres. It seems like this mapping isn't doing anything... if it is, where are the tests?

Reply:
I added it because I noticed we have `timestamp without timezone` but not the `with timezone` version.

And I didn't pay much attention to the space between `time` and `zone`, was just duplicating the  `timestamp without timezone` line and removing the `out` from `without` :D.

I guess JDBC always prefers to return `timestamp and timestamptz` instead of the long name version, so that's why our `timestamp without timezone` has been there for a while and there are no problems with it?

Anw, I'll add a space for both of them and leave it there.